fgpyo/sam/__init__.py

```diff
@@ -368,7 +368,7 @@ class CigarOp(enum.Enum):
         code (int): The `~pysam` cigar operator code.
         character (int): The single character cigar operator.
         consumes_query (bool): True if this operator consumes query bases, False otherwise.
-        consumes_target (bool): True if this operator consumes target bases, False otherwise.
+        consumes_reference (bool): True if this operator consumes reference bases, False otherwise.
```

Comment:
This is an unrelated fix. if deemed imporatant I could extract it to a separate PR.

Reply:
@yfarjoun This uses `target` intentionally - not all alignments are between a read and a reference - what if you align two reads vs. each other, or two contigs vs. each other?

Reply:
I thought I was correcting a broken link.... if this is intentional, I'll bring it back and let's haved the discussion in a different PR (if at all)

Reply:
100% on different PR.  Though I'll also argue there for maintaining "target".

Little known fact is that CIGAR actually predates SAM/BAM and Google believes it was first documented as part of [exonerate](https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate-manual) which uses "query" and "target" 😛